static/index.html

```diff
@@ -26,7 +26,7 @@
           <tbody id="manifest-gui-body"></tbody>
           <tfoot
             ><tr
-              ><td></td><td><button id="remove"></button><button id="add"></button></td></tr
+              ><td></td><td><button id="reset-to-default"></button><button id="remove"></button><button id="add"></button></td></tr
```

Comment:
If there is still time we should move buttons outside of table. Tables are terrible for layouting and will definitely generate padding/overflow problems eventually. I think the config itself is fine within table as there is no easy better way to do it.

Reply:
Move them to where? I lack the imagination lmao. Perhaps the little sprinkles and sparkles can be handled as one PR once this is stable? https://github.com/ubiquity-os/ubiquity-os-plugin-installer/issues/14 will be the last change to the UI for for V1